libcontainer/standard_init_linux.go

```diff
@@ -241,6 +241,7 @@ func (l *linuxStandardInit) Init() error {
 	// Close the pipe to signal that we have completed our init.
 	logrus.Debugf("init: closing the pipe to signal completion")
 	_ = l.pipe.Close()
+	l.pipe.procReady = true
```

Comment:
If change this to `l.pipe.closed`, this should be set inside `Close()` and there probably should be a helper method like `isClosed()` that is used by `startInitialization`.